hc/api/tests/test_pause.py

```diff
@@ -9,19 +10,26 @@ def test_it_works(self):
         check.save()
 
         url = "/api/v1/checks/%s/pause" % check.code
-        r = self.client.post(url, "", content_type="application/json",
+        resp = self.client.post(url, "", content_type="application/json",
                              HTTP_X_API_KEY="abc")
 
         ### Assert the expected status code and check's status
+        check.refresh_from_db()
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(check.status, 'paused')
 
     def test_it_validates_ownership(self):
         check = Check(user=self.bob, status="up")
         check.save()
 
         url = "/api/v1/checks/%s/pause" % check.code
-        r = self.client.post(url, "", content_type="application/json",
+        resp = self.client.post(url, "", content_type="application/json",
                              HTTP_X_API_KEY="abc")
 
-        self.assertEqual(r.status_code, 400)
+        self.assertEqual(resp.status_code, 400)
 
         ### Test that it only allows post requests
+        # Reference -->>> https://docs.djangoproject.com/en/dev/topics/testing/advanced/
+        req_factory = RequestFactory()
+        req = req_factory.post(url, "", content_type="application/json", HTTP_X_API_KEY="abc")
+        self.assertEqual(req.method, 'POST')
```

Comment:
@andela-uawili I get that. I could make a simple get request using the test client and check for a bad request status code.